src/lib/components/issue-card.svelte

```diff
@@ -37,6 +37,9 @@
         >
       </div>
     </div>
+    {#if `https://github.com/${issue.repository.owner.login}/${issue.repository.name}/blob/main/LICENSE`}
+      <span>True</span>
+    {/if}
```

Comment:
I think this change doesn't work unfortunately, you are just checking a string, and not the actual repo.

What you would need to do, is extend the GraphQL-Query and retrieve the data from the Github-API

Reply:
For testing the GraphQL-Query, you can use the Explorer from Github [here](https://docs.github.com/en/graphql/overview/explorer)

Reply:
yeah makes sense I just thought of trying this, thank you this will work !